src/System.Text.Json/tests/Serialization/Object.ReadTests.cs

```diff
@@ -180,14 +180,6 @@ public string Name
             }
         }
 
-        [Fact]
-        public static void ParseUntyped()
-        {
-            // Not supported until we are able to deserialize into JsonElement.
-            Assert.Throws<JsonException>(() => JsonSerializer.Parse<SimpleTestClass>("[]"));
```

Comment:
Can we keep this test case (non-System.Object).

Reply:
I am in the process of expanding the object tests; I'll move some of these tests to this PR.